lte/gateway/configs/mme.yml

```diff
@@ -20,7 +20,6 @@ cert_dir: "/usr/local/etc/oai"
 hss_ip: "192.168.60.153"
 hss_hostname: "hss"
 use_stateless: true
-enable_converged_core: false
```

Comment:
make sure to mention the removal of this local service config on the Summary section